### PR TITLE
Add loading property

### PR DESCRIPTION
**Pull request by RoXuS**
Hey !

Thx for your element !!

I just add a commit to display a loader while the token is created.

And an other commit to set displayError as a public method. Then, we will use the component to display the other errors from Stripe API.

### CODE REVIEW

**Comment by morbidick** [comment]
Hi,
thanks for your PR.
Just one thing, if we have `displayError` as public method we should add a check for [Line 254](https://github.com/RoXuS/stripe-elements/blob/e04e3707e049fe7c48b98033d7237ee19eb82f57/stripe-card.html#L254) to not run into any errors where `param` is set with no matching form element. Would you mind adding another commit?

**Comment by RoXuS** [comment]
Hi @morbidick,
Good catch, I have added the check, it is ok for you ?

File: stripe-card.html [hunk 1]
```diff
@@ -250,12 +266,14 @@
...
       displayError(error) {
         if (error.type == "card_error" && error.param) {
-          let element = this.$.form.querySelectorAll(`[name="${error.param}"]`)[0];
-          element.invalid = true;
+          let element = this.$.form.querySelector(`[name="${error.param}"]`);
 
-          if (error.param == "number") {
-            element.errorMessage = error.message;
-            return;
+          if (element) {
```
Comment (morbidick) [inline stripe-card.html:255]: the check looks good, nice change with `querySelector` i like it :+1:

File: stripe-card.html [hunk 2]
```diff
@@ -297,4 +315,4 @@
     }
     customElements.define(StripeCard.is, StripeCard);
   </script>
-</dom-module>
+</dom-module>
```
Comment (morbidick) [inline stripe-card.html]: A lot of whitespace got added with this commit, and this one removed, could you revert those changes?

**Review by morbidick — CHANGES_REQUESTED** [review]
Thank your for your commit, reverting the whitespace changes and we can get this merged! The travis build fail is my mistake, i planned to at a travis lint job for the next release as soon as this is merged.

**Comment by RoXuS** [comment]
Oh yeah sorry @morbidick, it's my IDE which auto format the code following my eslint conf ^^.

I rebase the last commit to avoid all whitespaces. It should be ok !

**Comment by morbidick** [comment]
Thank you very much @RoXuS, Im going to update `analysis.json` and push a new release.

**Pull request merged**